htdocs/comm/action/card.php

```diff
@@ -287,8 +287,8 @@
 		    $object->elementtype = 'task';
 		}
 
-		$object->datep = $datep;
-		$object->datef = $datef;
+		$object->datep = (int) $datep;
```

Comment:
$datep is result of dol_mktime that can return '' that is different than 0. Can you confirm we need this ?

Reply:
It was to avoid string into int, but will look at that later